src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_release.yaml

```
@@ -0,0 +1,68 @@
+post:
+  summary: Release sources and binaries of a specified package.
+  description: |
+    Release sources and binaries of the specified package. If no release target is specified through
+    the optional parameters, it will trigger the release for all manual set release targets
+    set on the specified project repository definitions for the specified package.
```

```suggestion
    the optional parameters, it will trigger the release for all manual release targets
    set on the specified project repository definitions for the specified package.
```
I'm not sure. Duplicated `set`?

---

Removed it @eduardoj